SA3/qa_web/urls.py

```diff
@@ -11,4 +11,6 @@
     url(r'^home/$', views.homepage, name='homepage'),
     path('vote/', views.vote, name='vote'),
     url(r'QuestionIndex/', views.QuestionDisplayView.as_view(), name='QuestionIndex'),
+    path('editprofile/', views.edit_profile, name='UserProfile'),
+    path('profile/<int:id_>/', views.display_profile, name='UserProfile'),
```

Comment:
I find it a little weird that anyone can access another's profile page without access restriction.

Reply:
are we spoke during our meeting on Friday February 23rd, we are enabling any user to view the other person's profile card.